### PR TITLE
[improve][pip] PIP-325: Add command to abort transaction

### PR DESCRIPTION
**Pull request by hrzzzz**
Releted PR: https://github.com/apache/pulsar/pull/21630

### Documentation

- [ ] `doc` 
- [ ] `doc-required` 
- [x] `doc-not-needed` 
- [ ] `doc-complete`

### CODE REVIEW

File: pip/pip-325.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+```
+pulsar-admin transactions abort-transaction --most-sig-bits 1 --least-sig-bits 2
+```
+### Metrics
+None.
+
+# Monitoring
+None.
+
+# Security Considerations
```
Comment (codelipenghui) [inline pip/pip-325.md:75]: What permission is required for the new admin API? The transaction owner? or super user?

Comment (hrzzzz) [reply pip/pip-325.md:75]: Is it okay if both the super user and the transaction owner are allowed to operate?

Comment (codelipenghui) [reply pip/pip-325.md:75]: Yes, it makes sense. Please also add it to the proposal.

Comment (hrzzzz) [reply pip/pip-325.md:75]: Added

Comment (hrzzzz) [reply pip/pip-325.md:75]: Should I start a vote for this PIP?

Comment (Technoboy-) [reply pip/pip-325.md:75]: yes, please start a vote for this PIP, the `discuss` has passed I think

**Comment by Technoboy-** [comment]
please start a vote for this PIP, the discuss has passed I think

**Comment by hrzzzz** [comment]
Mailing List discussion thread: https://lists.apache.org/thread/ssgngyrlgx36zvygvsd5b2dm5q6krn0f
Mailing List voting thread: https://lists.apache.org/thread/kp9w4d8drngomx1mdof0203ybgfmvtty

**Review by Technoboy- — APPROVED** [review]

**Pull request merged**